Readme.md

```diff
@@ -94,15 +94,17 @@ Notes:
 
 ## What You Can Expect to Sync
 * Position/Rotation
+* Position/Rotation when changed by Slew Mode
```

Comment:
I don't think this is needed as changing position/rotation in slew mode is still the same as position/rotation on its own